core/views.py

```diff
@@ -583,3 +589,37 @@ def get_context_data(self, **kwargs):
         ctx['results'] = results
 
         return ctx
+
+class OpenJobListView(APIView):
+    def get(self, request):
+        # Parameters recieving
+        limit = int(request.query_params.get('limit', 10))
+        offset = int(request.query_params.get('offset', 0))
+
+        url = "https://jobs.ucf.edu/jobs/search"
+        response = requests.get(url)
+        soup = BeautifulSoup(response.content, 'html.parser')  # Specify the parser
+
+        cards = soup.find_all(class_="job-search-results-card-title")
+
+        # Base URL to remove
+        base_url = "https://jobs.ucf.edu/jobs"
```

Comment:
This needs to be a configuration item. It doesn't need to be changed this pull request, but it does need to happen before this feature is released.

Reply:
it was modified.